spinc/cmd/find.go

```diff
@@ -255,6 +268,7 @@ Filters:
   type        type of request to return
   states      comma-separated list of request states to include
   user        return only requests made by this user
+  requestArgs return requests made with specific args (format: arg1=value1,arg2=value2)
```

Comment:
`requestArgs` -> `args` I think, cause you're grabbing "args" from the map at line 108

Reply:
done